jctools-experimental/src/main/java/org/jctools/channels/OffHeapFixedMessageSizeRingBuffer.java

```diff
@@ -33,10 +33,13 @@
  */
 public abstract class OffHeapFixedMessageSizeRingBuffer extends ProxyChannelRingBuffer {
 
-    public static final int READ_RELEASE_INDICATOR = 0;
-    public static final int READ_ACQUIRE_INDICATOR = 1;
-    public static final int WRITE_RELEASE_INDICATOR = 2;
-    public static final int WRITE_ACQUIRE_INDICATOR = 3;
+    /*
+     * Valid values in the buffer are >0, so we use <0 sentinel values for
+     */
```

Comment:
This isn't currently an issue with the existing SPSC and MPSC implementations but could easily be overlooked in other variations, so I've just preempted the issue. Perhaps unnecessarily but I don't see the harm.